core/domain/user_services.py

```diff
@@ -28,6 +31,11 @@
 (user_models,) = models.Registry.import_models([models.NAMES.user])
 
 MAX_USERNAME_LENGTH = 50
+# Size (in px) of the gravatar being retrieved.
+GRAVATAR_SIZE = 150
+# Data url for images/avatar/user_blue_72px.png
+DEFAULT_IDENTICON_DATA_URL = \
```

Comment:
We never use backslashes in the codebase for this purpose. The reason for this is because it's hard to tell what is being escaped -- it may be a trailing space, for example. Instead, please use, e.g., parentheses.